packages/@aws-cdk/aws-appmesh/lib/virtual-gateway.ts

```diff
@@ -173,15 +174,19 @@ export class VirtualGateway extends VirtualGatewayBase {
     }
 
     const accessLogging = props.accessLog?.bind(this);
+    const tlsClientPolicy = props.backendDefaults?.tlsClientPolicy;
 
     const node = new CfnVirtualGateway(this, 'Resource', {
       virtualGatewayName: this.physicalName,
       meshName: this.mesh.meshName,
       spec: {
         listeners: this.listeners.map(listener => listener.listener),
-        backendDefaults: props.backendDefaults !== undefined
+        backendDefaults: props.backendDefaults
```

Comment:
I don't think there's any need for this change, I would leave it as-is.

Reply:
I am ok leaving `! == undefined` but wouldn't this be simply redundant?

Reply:
It is redundant, but that just means there's little point in changing it. Generally, the smaller the PR, the better 🙂. That's the only motivation I have here.

Reply:
Makes sense! I agree `!== undefined` is not a necessary change and thus leaving it as-is to keep the PR smaller.